superset/assets/javascripts/explorev2/reducers/exploreReducer.js

```diff
@@ -90,6 +87,7 @@ export const exploreReducer = function (state, action) {
           chartUpdateEndTime: null,
           chartUpdateStartTime: now(),
           triggerQuery: false,
+          queryRequest: action.queryRequest,
```

Comment:
i'm don't think this is problematic. i'm doing a little research to see how others are handling this, but i don't know of a reason why this wouldn't be ok at this point.